transport/server.cc

```diff
@@ -188,9 +188,13 @@ event::event_type parse_event_type(const sstring& value)
     }
 }
 
-cql_sg_stats::cql_sg_stats()
+cql_sg_stats::cql_sg_stats(bool enable_maintenance_port)
     : _cql_requests_stats(static_cast<uint8_t>(cql_binary_opcode::OPCODES_COUNT))
 {
+    if (enable_maintenance_port) {
+        return;
+    }
+
```

Comment:
I don't understand why maintenance mode has to leak into something like cql statistics.

We don't need to split statistics for maintenance mode, but we can share the counters.